### DISCUSSION TITLE
Received Message Larger than Max Issue

### DISCUSSION THREAD

**Discussion by piyush-coschool** [apache/skywalking#12701] [Q&A]
Using skywalking-backend-js for nodejs, getting

code: 8,
details: 'Received message larger than max (1952543861 vs 4194304)',
metadata: Metadata { internalRepr: Map(0) {}, options: {} },
level: 'error'

Using this:
`import Agent from 'skywalking-backend-js';
Agent.start({
  serviceName: process.env.SERVICE_NAME ,
  serviceInstance: process.env.SERVICE_INSTANCE,
  collectorAddress: process.env.SKYWALKING_URL,
});`

**Comment by wu-sheng** [comment]
Could you attach the screenshot? Is this a special page or some cases?

**Reply by piyush-coschool** [reply]
Just added the above code to main.ts
![Screenshot 2024-10-16 at 7 12 21 PM](https://github.com/user-attachments/assets/bfce2b22-6123-4732-865b-f35459adf20f)

**Reply by wu-sheng** [reply]
@Fine0830 Any idea?

**Reply by kezhenxu94** [reply]
> Just added the above code to main.ts
> 
> ![Screenshot 2024-10-16 at 7 12 21 PM](https://github.com/user-attachments/assets/bfce2b22-6123-4732-865b-f35459adf20f)

Can it be that the traces data is too large? For example too many spans in the trace?

**Reply by piyush-coschool** [reply]
@kezhenxu94 I have not added any span as of now, currently everything is in default configuration.
